Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -414,31 +414,7 @@ function CreateDiscussionThreads
         $discussionComment.IsDeleted = $false;
         $discussionComment.Content = $message.Content
 
-        $properties = New-Object -TypeName "Microsoft.VisualStudio.Services.WebApi.PropertiesCollection"
-        
-        if ($script:pullRequest.CodeReviewId -gt 0)
-        {
-            $changeTrackingId = 0;
-            if (!(TryGetCodeFlowChangeTrackingId($changes, $message.RelativePath, [Ref]$changeTrackingId)))
```

Comment:
The bug is here. In PS calling a function using paranthesis is bad - the paranthesis and commas is used to specify an Array param. See http://stackoverflow.com/questions/7149783/powershell-pass-by-reference-not-working-for-me for more details. This regression was introduced by a change I made ~5 weeks ago.